test/Liquidator/Liquidator.t.sol

```diff
@@ -134,14 +129,13 @@ contract LiquidatorTest is Test {
         );
```

Comment:
Since the amounts are pretty specific here we don't really need to make these tests more generic, however, can you add another tests that specifically tests a GHO liquidation? So it would be borrow GHO against ETH or something and then run a liquidation by flash borrowing from Balancer or something.

Reply:
One thing to be mindful here is that when we run a liquidation on chain we generally use 0x to execute the swap. However, if we are flash borrowing from 0x we do need to explicitly exclude Balancer V2 from the swap or we will fail on re-entrancy.